app/questionnaire/placeholder_transforms.py

```diff
@@ -183,3 +183,8 @@ def first_non_empty_item(self, items):
             return item
 
         return ""
+
+    def format_suggestions_url(self, url, list_json):
+        if self.language == "en":
+            return url + list_json
```

Comment:
We should use .format() to concatenate strings.

Reply:
Done, thanks👍